.github/workflows/twig-cs-fixer.yml

```
@@ -0,0 +1,50 @@
+name: Twig CS Fixer
+
+on:
+  push:
+    paths:
+      - '**.twig'
+  pull_request:
+    paths:
+      - '**.twig'
+
+concurrency:
+    group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+    cancel-in-progress: true
+
+permissions:
+  contents: read
+
+jobs:
+  twig-cs-fixer:
+    name: Twig CS Fixer
+    runs-on: ubuntu-24.04
+
+    env:
+      php-version: '8.4'
+    steps:
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ env.php-version }}
+          ini-values: "memory_limit=-1"
+          coverage: none
+
+      - name: Checkout target branch
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.base_ref }}
+
+      - name: Checkout PR
+        uses: actions/checkout@v4
+
+      - name: Install dependencies
+        run: |
+          COMPOSER_HOME="$(composer config home)"
+          ([ -d "$COMPOSER_HOME" ] || mkdir "$COMPOSER_HOME") && cp .github/composer-config.json "$COMPOSER_HOME/config.json"
+          export COMPOSER_ROOT_VERSION=$(grep ' VERSION = ' src/Symfony/Component/HttpKernel/Kernel.php | grep -P -o '[0-9]+\.[0-9]+').x-dev
+          composer require --dev vincentlanglet/twig-cs-fixer:^3.7
```

```suggestion
          composer require --dev vincentlanglet/twig-cs-fixer:^3.8.1
```
Use the latest compatible version: https://github.com/symfony/symfony/pull/60761#discussion_r2153897739

---

obviously !